### PR TITLE
Move all textures to common folder / Clean-up textures

### PR DESCRIPTION
**Pull request by croniccorey**
I am not sure if this change will be of interest but this moves all images into a common folder and removes any images that are not used.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -43,18 +43,13 @@ $(BUILDDIR)/$(ADDON_NAME)/resources: $(BUILDDIR)/$(ADDON_NAME)/resources/skins \
...
-$(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media/default:
-	mkdir -p $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media/default
-	cp textures/$(DISTRONAME)/*.{png,jpg} $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media/default
-
-$(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media/icons:
-	mkdir -p $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media/icons
-	cp icons/*.png $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media/icons
+$(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media:
+	mkdir -p $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media
+	cp textures/$(DISTRONAME)/*.{png,jpg} $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media
```
Comment (MilhouseVH) [inline Makefile]: We also need:
```
cp icons/*.png $(BUILDDIR)/$(ADDON_NAME)/resources/skins/Default/media
```

Comment (MilhouseVH) [reply Makefile]: Ah OK maybe not, perhaps I should read all the changes before commenting... :)

**Comment by MilhouseVH** [comment]
This all seems fine - I'd like to include in nightly builds for a few days to be sure.

**Comment by MilhouseVH** [comment]
Not related to these changes, but in the same area - the following two lines reference `$VAR` variables that don't appear to be valid/set (at least, not for LibreELEC):

https://github.com/LibreELEC/service.libreelec.settings/blob/80323b6c36ceef60eefe0ae3b3cbc9283a63cdaa/skins/Default/1080i/service-LibreELEC-Settings-mainWindow.xml#L13

https://github.com/LibreELEC/service.libreelec.settings/blob/80323b6c36ceef60eefe0ae3b3cbc9283a63cdaa/skins/Default/1080i/service-LibreELEC-Settings-mainWindow.xml#L21

I'm not sure how they can become set - is this something that the current skin can set (Kodi core doesn't seem to include any references to these variables).

The first currently produces the following WARNING:
```
2019-03-25 11:19:07.885 T:1936667968 WARNING: Label Formating: $VAR[GlobalFanartVar] is not defined
```
and I'm assuming the second just fails silently.

@croniccorey @edit4ever (or anyone else) do you know if we can fix these, presumably they're no longer required and we should remove the reference to `$VAR[GlobalFanartVar]` and delete the second line completely?

**Comment by croniccorey** [comment]
i have removed the line in my builds with no issue, not sure what it is for but it is obsolete i think. i can include the commit if you want.

**Comment by MilhouseVH** [comment]
@croniccorey yes please add the commit, I'll get it tested in the nightlies but don't think it will be an issue.

I suspect we can drop the second one too.

**Comment by croniccorey** [comment]
I did not notice the second one since i never seen a warning for it but the first one is not needed from what i can tell.

**Comment by croniccorey** [comment]
I did not change anything here just removed 2 images i missed, i will not make any more changes.

**Pull request merged**